dsf/org.eclipse.cdt.dsf.ui/src/org/eclipse/cdt/dsf/debug/ui/actions/DsfSteppingModeTarget.java

```diff
@@ -26,23 +26,21 @@
  *
  * @since 1.0
  */
-public class DsfSteppingModeTarget implements ISteppingModeTarget, ITargetProperties {
+public class DsfSteppingModeTarget implements ISteppingModeTarget {
 
 	private static final String ID_DISASSEMBLY_VIEW = "org.eclipse.cdt.dsf.debug.ui.disassembly.view"; //$NON-NLS-1$
 
-	private final Preferences fPreferences;
+	private IEclipsePreferences fPreferences = InstanceScope.INSTANCE.getNode("org.eclipse.cdt.dsf.ui"); //$NON-NLS-1$
```

Comment:
This is just a suggestion, feel free do dismiss.
```suggestion
	private IEclipsePreferences fPreferences = InstanceScope.INSTANCE.getNode(CDebugUIPlugin.PLUGIN_ID);
```

Reply:
Fixed.